IFC4x3/Properties/u/UndercontrolEquipmentsID_0uZsrOMH5AMBQLhPS3ukXv/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="UndercontrolEquipmentsID_0uZsrOMH5AMBQLhPS3ukXv" Name="UndercontrolEquipmentsID" UniqueId="388f6d58-5911-4a58-b695-ad9703e2e879" PropertyType="p_listvalue" PrimaryDataType="IfcIdentifier" />
```

Comment:
As `IfcUnitaryControlElement` is an assembly of control elements, why not just use an aggregation to determine this? Seems like overkill here.

Reply:
Same as above